UIforETW/ChildProcess.cpp

```diff
@@ -112,10 +114,11 @@ DWORD ChildProcess::ListenerThread()
 	return 0;
 }
 
-
+_Pre_satisfies_( hProcess_ == 0 )
+_Success_( return )
 bool ChildProcess::Run(bool showCommand, std::wstring args)
 {
-	assert(!hProcess_);
+	ATLASSERT(!hProcess_);

```

Comment:
What's the advantage of ATLASSERT? The default assert macro is pretty crappy (reentrancy problems in the UI) but is ATLASSERT any better?